zproject/default_settings.py

```diff
@@ -199,29 +199,47 @@
 EMBEDDED_BOTS_ENABLED = False
 
 DEFAULT_RATE_LIMITING_RULES = {
+    # Applies to API requests in authenticated contexts,
+    # each user gets their own bucket.
     "api_by_user": [
         (60, 200),  # 200 requests max every minute
     ],
+    # Applies to API requests in unauthenticated contexts
+    # (web public views) - ONLY if the request is not authenticated.
     "api_by_ip": [
         (60, 100),
     ],
+    # Applies to requests to our push bouncer API, by an authenticated
+    # remote server.
```

Comment:
"our" hints at this, but we can probably make a bit clearer that this is only relevant for locations which are running their own push bouncer service -- it's not used when talking _to_ the push bouncer.

Reply:
Yup, good thought. Struggled a bit with phrasing this because the fact that "push notification bouncer service" is a completely different thing from just the normal server API that handles requests from mobile users for registering the device etc. is probably not obvious from the naming. Went with something like this to try to make it clear that this is almost certainly not relevant for the server admin:
```
    # Applies to requests to the push bouncer service API, by an authenticated
    # remote server. This is only applicable if the server is running the
    # push bouncer service for other servers to send push notifications through,
    # which is most likely just the Zulip Cloud server.
```